core/stse_frame.c

```diff
@@ -311,8 +311,12 @@ stse_ReturnCode_t stse_frame_receive(stse_Handler_t* pSTSE, stse_frame_t* pFrame
     }
 
     /* - Verify correct reception*/
-    if((ret & 0x3F)!= STSE_OK)
+    if((ret & 0x1F)!= STSE_OK)
```

Comment:
we should use a definition here . Proposal : 
#define STSE_RSP_STATUS_MASK 0x1F

Reply:
Done